app/components/at_mention/at_mention.js

```diff
@@ -152,6 +152,13 @@ export default class AtMention extends React.PureComponent {
         let onPress;
         let suffix;
         let suffixElement;
+        let styleText;
+
+        if (textStyle) {
+            const {backgroundColor: bg, ...otherStyles} = StyleSheet.flatten(textStyle);
```

Comment:
I suppose not storing the entire `backgroundColor` object in `backgroundColor` variable is okay? (Since previously we weren't only taking the `bg` property but rather the entire `backgroundColor` object)

Reply:
the backgroundColor has never been an object but  a string ;) this syntax `{backgroundColor: bg} = someObject` basically takes the value of `backgroundColor` and sets it in the `bg` variable name.

Reply:
see https://www.barrymichaeldoyle.com/destructuring/